le_utils/validators/common.py

```diff
@@ -11,9 +11,11 @@ def get_embed_schema_resolver(schema):
     :param schema: The schema to use as the base schema
     :return: A configured RefResolver
     """
-    schema_dir = os.path.join(os.path.dirname(__file__), "..", "..", "spec")
-    with open(os.path.join(schema_dir, "definitions-embed_common.json")) as f:
-        common_definitions = json.load(f)
+    common_definitions = json.loads(
+        pkgutil.get_data("le_utils", "resources/embedcommondefinitions.json").decode(
```

Comment:
I kinda liked the old file name, because it followed the pattern of the existing files. Did it contain disallowed characters?

Reply:
I don't think so. I guess it was more of a convention change in the context of the 'resources' folder. Will make the change and test it out.

Reply:
Made the change--works correctly.